packages/react/src/components/Skeleton/Text/Text.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { type HTMLAttributes } from 'react';
+import cn from 'classnames';
+
+import classes from '../Skeleton.module.css';
+import { useSynchronizedAnimation } from '../../../hooks';
+
+export type TextProps = {
+  width?: string | number;
+  height?: string | number;
+} & HTMLAttributes<HTMLDivElement>;
+
+/**  Skeleton component used for indicating loading elements of circular shape */
+export const Text = ({
+  width,
+  height,
+  className,
+  children,
+  ...rest
+}: TextProps) => {
+  const ref = useSynchronizedAnimation('opacity-fade');
+
+  return (
+    <div
+      ref={ref}
+      className={cn(classes.skeleton, classes.text, className, {
+        [classes.hasChildren]: Boolean(children),
+      })}
+      style={{ width, height }}
+      aria-hidden
+      {...rest}
```

Comment:
You need to pass `rest.style` and `rest.className` in their respective props, and put `{...rest}` above them, otherwise everything we have set will be overwritten.

Reply:
Good catch!
Fixed! 😊